fairseq/tasks/translation.py

```diff
@@ -492,6 +492,8 @@ def decode(toks, escape_unk=False):
             logger.info("example hypothesis: " + hyps[0])
             logger.info("example reference: " + refs[0])
         if self.cfg.eval_tokenized_bleu:
-            return sacrebleu.corpus_bleu(hyps, [refs], tokenize="none")
+            return sacrebleu.metrics.BLEU().corpus_score(hyps, [refs])
```

Comment:
I don't think this is correct. IIUC this will now use the default Sacrebleu tokenizer instead of using the model tokenizer.

Reply:
The default parameter of `BLEU()` is as followd:
```py
    def __init__(self, lowercase: bool = False,
                 force: bool = False,
                 tokenize: Optional[str] = None,
                 smooth_method: str = 'exp',
                 smooth_value: Optional[float] = None,
                 max_ngram_order: int = MAX_NGRAM_ORDER,
                 effective_order: bool = False,
                 trg_lang: str = '',
                 references: Optional[Sequence[Sequence[str]]] = None):
```
which is same as compatiable api `sacrebleu.corpus_bleu` with setting `tokenize="none"`
```
def corpus_bleu(hypotheses: Sequence[str],
                references: Sequence[Sequence[str]],
                smooth_method='exp',
                smooth_value=None,
                force=False,
                lowercase=False,
                tokenize=BLEU.TOKENIZER_DEFAULT,
                use_effective_order=False) -> BLEUScore:
```